### ISSUE TITLE
Accessibility checks

### ISSUE THREAD

**Issue by kw699** [MattyP15/Ai-Failure-Museum#11]
conduct and document accessibility checks 
[ link to WCAG compliance page https://www.audioeye.com/compliance/wcag/ ]
